index.js

```diff
@@ -1,11 +1,17 @@
+
+Yeah im already feeling super discouraged so sorry for submitting this late 
```

Comment:
It's all good, you'll have time this week to get in shape with Javascript - just make sure to keep up and practice, you'll get there 💪